tests/csm/rest/test_csm_load.py

```diff
@@ -277,6 +293,130 @@ def test_44257(self):
         self.log.info("##### Test completed -  %s #####", test_case_name)
 
 
+    # pylint: disable=too-many-statements
+    @pytest.mark.skip(reason="not_in_main_build_yet")
+    @pytest.mark.lc
+    @pytest.mark.jmeter
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44799')
+    def test_44799(self):
+        """
+        Verify max number of IAM users using all the required parameters
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.test_cfgs["test_44799"]
+        jmx_file = "CSM_create_IAM_user_Loaded.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+
+        resp = self.csm_obj.list_iam_users_rgw()
+        assert resp.status_code == HTTPStatus.OK, "List IAM user failed."
+        user_data = resp.json()
+        self.log.info("Step 1: List user response : %s", user_data)
+        existing_user = len(user_data['users'])
+        self.log.info("Existing iam users count: %s", existing_user)
+        self.log.info("Max iam users : %s", rest_const.MAX_IAM_USERS)
+        new_iam_users = rest_const.MAX_IAM_USERS - existing_user
+        self.log.info("New users to create: %s", new_iam_users)
+        thread_request = test_cfg["thread_request"] # ( login + create )
+        operation_count = new_iam_users * thread_request
+
+        request_usage_rounded = int( thread_request * round( self.request_usage / thread_request ))
+        loop = operation_count // request_usage_rounded
+        req_in_loops = request_usage_rounded * loop
+        operation_last = operation_count - req_in_loops
+        threads_1 = request_usage_rounded // thread_request
+        threads_2 = operation_last // thread_request
```

Comment:
@kachhwahadivya have a lib function to do this less complicated and clean, will be updated later